GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1170,41 +1170,10 @@ string BuildPathFilter(string? path)
                 };
 
                 HashSet<string?> setOfFileNames = new();
-                ExecutionResult result = Module.GitExecutable.Execute(args, outputEncoding: GitModule.LosslessEncoding, throwOnErrorExit: false);
-                var lines = result.StandardOutput.LazySplit('\n');
 
-                // TODO Check the exit code and warn the user that rename detection could not be done.
-
-                ObjectId currentObjectId = null;
-                foreach (var line in lines.Select(GitModule.ReEncodeFileNameFromLossless))
+                foreach (var item in ParseFileNames(args, startOfObjectId))
```

Comment:
MC
```suggestion
                foreach (string item in ParseFileNames(args, startOfObjectId))
```

Reply:
`ParseFileNames` no longer returns a `string` but a pair of strings. Should I deconstruct it like
`foreach ((_, string fileName) in ParseFileNames(args, startOfObjectId))`?

Reply:
Yes, please

Reply:
A gentle ping